docs/assets/js/swagger.yml

```diff
@@ -1258,54 +1287,6 @@ paths:
           schema:
             $ref: "#/definitions/error"
 
-  /repositories/{repository}/inventory/s3/import:
```

Comment:
This file is "auto-generated" by `make build` (in the sense of "blindly copied because Go and WebPack each have their own uniquely correct view of the world, but that view is hopelessly naïve and wrong so we have to hack our away around it because explicit dependencies are very 1990s".  @nopcoder can explain why it is committed.
The last person to touch swagger.yml just forgot to build it.
Mine is an exact copy:
```
ariels@ariels:~/Dev/lakeFS$ diff swagger.yml docs/assets/js/swagger.yml 
```